plugins/builds/update.js

```diff
@@ -86,28 +87,13 @@ module.exports = () => ({
                                 return null;
                             }
 
-                            const workflow = pipeline.workflow;
+                            const workflowGraph = pipeline.workflowGraph;
```

Comment:
is this coming form event ? We would probably want to retrieve current workflow from event which this build is part of. I also think that event should have the status of all builds (jobs), so that when use case like fan-in is implemented, this API will look at status of workflowGraph in event to figure out what needs to be triggered as part of workflow for the event.

Reply:
Good point, will fix it.